python-scripts/scripts/visualisze.py

```diff
@@ -0,0 +1,149 @@
+'''Data preparatie en visualisaite defenities'''
```

Comment:
kies 1 van deze 2 voor naam van functie: visualise of visualize (GB/US spelling)